src/AbstractOperations/computed_field.jl

```diff
@@ -89,3 +89,62 @@ end
 
     @inbounds data[i′, j′, k′] = operand[i′, j′, k′]
 end
+
+struct FusedComputedFields
+    fields::Vector{ComputedField}
+    grid::AbstractGrid
```

Comment:
This needs to be type parameterized, otherwise `FusedComputedFields` is not concretely typed:

```julia
grid :: G
```